src/PooledArrays.jl

```diff
@@ -376,9 +376,8 @@ end
 ##############################################################################
 
 function Base.push!(pv::PooledVector{S,R}, v::T) where {S,R,T}
-    v = convert(S,v)
```

Comment:
`v` is still needed. Same below.

Reply:
Why?
We alraedy have `v` ?
the `push!(pv.refs, v)` will convert internally